docs/topics/request-response.rst

```diff
@@ -364,6 +364,18 @@ started, i.e. HTTP message sent over the network. This meta key only becomes
 available when the response has been downloaded. While most other meta keys are
 used to control Scrapy behavior, this one is supposed to be read-only.
 
+.. reqmeta:: download_slot
+
+download_slot
+----------------
+
+The website domain to which the request is destined. It is used to provide info
+when :setting:`AUTOTHROTTLE_ENABLED` is set to `True`. This meta key only becomes
+available when the response has been downloaded. While most other meta keys are
+used to control Scrapy behavior, this one is supposed to be read-only.
+See also: :ref:`topics-autothrottle` and its
+:setting:`AUTOTHROTTLE_ENABLED` option.
```

Comment:
Hey! Unfortunately this explanation is not correct; this meta key is not related to AutoThrottle directly, it is a general Downloader thing, and it is not supposed to be read-only - it allows to change a default slot for a domain, and tune concurreency options this way.

I'd suggest to either study how this slot is used, by checking Downloader source code, and maybe some examples (e.g. https://github.com/scrapy-plugins/scrapy-splash), to really understand what is this option for, or try helping Scrapy in some other way - download_slot is an advanced feature, which is not that easy to document.

Reply:
Hey @kmike Thanks for pointing out! I already looked at the `Downloader` source code, I understand that it uses hostname as slot when dealing with concurrency. That key has little usage in the actual project, that's it's actual purpose wasn't clear. With the example you provided I'll explore the usage and come up with a better explanation for this key.

Reply:
HI @kmike, can you please have a look? Any feedback is welcomed.